barretenberg/cpp/src/barretenberg/vm2/tracegen/lib/lookup_builder.hpp

```diff
@@ -30,12 +30,11 @@ template <typename LookupSettings_> class BaseLookupTraceBuilder : public Intera
         // find a row dst_row in the target columns {d1, d2, ...} where the values match.
         // Then we increment the count in the counts column at dst_row.
         // The complexity is O(|src_selector|) * O(find_in_dst).
-        trace.visit_column(LookupSettings::SRC_SELECTOR, [&](uint32_t row, const FF& src_sel_value) {
-            assert(src_sel_value == 1);
```

Comment:
I've decided this is kind of testing that visit_column works ok, so it's out of place here